Mathlib/Algebra/Quaternion.lean

```diff
@@ -64,45 +64,49 @@ structure QuaternionAlgebra (R : Type*) (a b : R) where
   imK : R
 
 @[inherit_doc]
-scoped[Quaternion] notation "ℍ[" R "," a "," b "]" => QuaternionAlgebra R a b
-open Quaternion
+scoped[Quaternion] notation "ℍ[" R "," a "," b "," c "]" =>
+    QuaternionAlgebra R a b c
+
+@[inherit_doc, deprecated "Use ℍ[R,a,0,b] instead of ℍ[R,a,b]." (since := "2025-01-15")]
```

Comment:
Is the syntax highlighting broken in vscode for the rest of this file too? If so, does adding `-- " this comment fixes the highlightin` fix it?